src/appshell/qml/Preferences/AppearancePreferencesPage.qml

```diff
@@ -142,5 +206,13 @@ PreferencesPage {
                 appearanceModel.foregroundWallpaperPath = newWallpaperPath
             }
         }
+
+        FlatButton {
+            text: qsTrc("appshell", "Reset Theme to Default")
```

Comment:
same

Reply:
Converted to `"Reset theme to default"`
Fixed.